### ISSUE TITLE
How to set hyper-parameters for FSSD training?

### ISSUE THREAD

**Issue by JesseYang** [lzx1413/PytorchSSD#9]
To achieve the mAP of 77.8, which is claimed in README, how to set hyper-parameters for FSSD? Or just use the default values in train_test.py?

The README also mentioned that there is a train_RFB.py which lists parameters for RFB training, but I did not find such a script in this repo

**Comment by lzx1413** [comment]
The hyper-parameter of FSSD 300 is the same with RFB300 except that the learning rate is 0.001

**Comment by JesseYang** [comment]
And the parameters for RFB300 are the default values in train_test.py? I didn't find the train_RFB.py mentioned in Readme file.

**Comment by lzx1413** [comment]
You are right, I merged the train-RFB.py and test-RFB.py into train_test.py

**Comment by JesseYang** [comment]
Thanks！
